### DISCUSSION TITLE
Windows11 没有通知窗口

### DISCUSSION THREAD

**Discussion by OxygenLost** [Jeric-X/SyncClipboard#83] [General]
是我哪里配置有问题吗
{
  "ServerService": {
    "SwitchOn": false,
    "Port": 5033,
    "UserName": "admin",
    "Password": "admin",
    "ClientMixedMode": false
  },
  "SyncService": {
///
///
///
    "SyncSwitchOn": true,
    "PullSwitchOn": true,
    "PushSwitchOn": true,
    "UseLocalServer": false,
    "DeletePreviousFilesOnPush": true,
    "NotifyOnDownloaded": true,
    "TrustInsecureCertificate": false,
    "MaxFileByte": 20971520,
    "IntervalTime": 1,
    "RetryTimes": 3,
    "TimeOut": 100
  },
  "Program": {
    "Proxy": "",
    "DeleteTempFilesOnStartUp": true,
    "LogRemainDays": 8,
    "CheckUpdateOnStartUp": true,
    "CheckUpdateForBeta": false,
    "Language": "",
    "Font": "",
    "HideWindowOnStartup": true,
    "DiagnoseMode": false,
    "DiagnosePageAutoRefresh": false,
    "Theme": ""
  }
}

**Comment by Jeric-X** [comment]
通知窗口指的是……？
看设置是没什么问题(NotifyOnDownloaded = true)，正常情况是在获取到新的剪贴板内容时在右下角弹出更新通知（图一），过几秒后收起到通知中心（图二），在勿扰模式下或者已经有其他弹出的通知时，会直接收进通知中心
如果确定没有通知的话，可能是程序BUG
![u4rp4q1t v2k](https://github.com/Jeric-X/SyncClipboard/assets/10058586/6c575691-edf7-481d-979a-f1bf90e53994)

![v1fftn1m 04k](https://github.com/Jeric-X/SyncClipboard/assets/10058586/b64fe89d-c285-4cf1-a9a6-3a193214de61)

**Comment by OxygenLost** [comment]
对 预期结果是这样。 但是我没有任何通知窗口。用的软件是 windows 最新版

JericX ***@***.***>于2024年6月4日 周二00:56写道：

**Comment by axutongxue** [comment]
解决办法是需要去windows设置中打开「获取来自应用和其他发送者的通知」
![20240816220833](https://github.com/user-attachments/assets/95bb2906-a6c2-43d7-969d-53afe4e47e57)
